client/sites-dashboard/components/sites-table-row.tsx

```diff
@@ -205,11 +210,18 @@ export default memo( function SitesTableRow( { site }: SiteTableRowProps ) {
 						</ListTileTitle>
 					}
 					subtitle={
-						<ListTileSubtitle>
-							<SiteUrl href={ siteUrl } title={ siteUrl }>
-								<Truncated>{ displaySiteUrl( siteUrl ) }</Truncated>
-							</SiteUrl>
-						</ListTileSubtitle>
+						<>
+							<ListTileSubtitle>
+								<SiteUrl href={ siteUrl } title={ siteUrl }>
+									<Truncated>{ displaySiteUrl( siteUrl ) }</Truncated>
+								</SiteUrl>
+							</ListTileSubtitle>
+							<ListTileSubtitle>
+								<SiteAdminLink href={ dashboardUrl } title={ __( 'Visit Dashboard' ) }>
```

Comment:
If the site's admin interface is Default, this links to My Home instead of `/wp-admin`, which is misleading. Is this expected? 😄

Reply:
I forgot it may point to My Home 😓